README.md

```diff
@@ -4,6 +4,9 @@ IDD is a tool for performing interactive dynamic differential debugging capable
 
 IDD loads two versions of the same application. The first one is the base version that works as expected while the second version of the same program has a regression introduced. IDD inspects the two versions of the applications using external tools like gdb and lldb. The two applications are executed side by side and the user is allowed to dispatch commands to the underlying debuggers in order to expect their internal states and isolate the origin of the regression.
 
+## Demo
+![idd](https://github.com/mvassilev/idd/assets/7579600/605ee84c-9d2d-4557-9290-59b384f4f848)
```

Comment:
The url should probably be on the main repository, not the fork. Maybe we should improve the commit message.